android/source/VideoLocker/src/org/edx/mobile/view/LoginActivity.java

```diff
@@ -62,17 +62,15 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
-        overridePendingTransition(R.anim.slide_in_from_right,
-                R.anim.slide_out_to_left);
+        overridePendingTransition(R.anim.slide_in_from_bottom, R.anim.no_transition);
 
-        getWindow().setSoftInputMode(
-                WindowManager.LayoutParams.SOFT_INPUT_STATE_HIDDEN);
+        hideSoftKeypad();
```

Comment:
Yes, but as <code>LoginActivity</code> has input views for email and password. Email input view takes the focus and displays keypad by default as screen is launched.
We just let the keypad be hidden by default.